### PR TITLE
1156334 restyle fxos

### PR DESCRIPTION
**Pull request by willkg**
Yeah, so that happened.

r?

### CODE REVIEW

**Comment by willkg** [comment]
This cleans up a bunch of stuff.

There's still a lot of duplicate js code between the generic and fxos forms, but they're pretty different under the hood because they had different requirements, so I'm going to leave it for now.

This passes the unit tests and also the smoke tests.

Reviewing this might be hard because the diffs are a mess.

To review:

1. look at the final code and comment on issues there
2. do generic and fxos feedback looking for flow issues with back-and-forth using browser back button and the form's back button
3. look at the final css for sanity
4. look at cards.js -- does that look ok?

File: fjord/feedback/static/js/cards.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+ *
+ * - all cards are divs with class "card"
+ * - inactive cards have the "inactive" class; when starting out
+ *   all cards should be marked inactive but the first one
+ * - the back button has the "back-button-container" class
+ * - data-focus attribute on the card has the id of the element
+ *   that should receive focus
+ * - header -> h1 gets the value of the data-title attribute on
+ *   the card
+ */
```
Comment (willkg) [inline fjord/feedback/static/js/cards.js]: This sucks. It should probably be a class where the above things are configurable, but I was trying to minimize the number of changes I was doing so I opted to document it for now and maybe fix it later.

File: fjord/settings/base.py [hunk 2]
```diff
@@ -650,7 +637,7 @@ def JINJA_CONFIG():
 ## Logging
 LOG_LEVEL = logging.INFO
 HAS_SYSLOG = True
-SYSLOG_TAG = "http_app_playdoh"  # Change this after you fork.
+SYSLOG_TAG = 'http_app_playdoh'  # Change this after you fork.
```
Comment (willkg) [inline fjord/settings/base.py]: I think the pre-commit hooks did this.

Comment (mythmon) [inline fjord/settings/base.py]: Seems likely: https://github.com/mozilla/fjord/blob/c115b7a14e06d16ebd06958b9656ec5bb5eb1f59/.pre-commit-config.yaml#L12

**Comment by willkg** [comment]
I need to rebase against master to pick up the footer width fix.

**Comment by willkg** [comment]
^^^ That folds in the footer width fix I just landed. Plus it nixes a bunch of cleanup commits. We can do those in another PR. Hopefully that reduces some of the review work.

File: fjord/feedback/static/js/fxos_feedback.js [hunk 3]
```diff
@@ -1,130 +1,173 @@
...
+        }
+        $('#happy-button').click(function(ev) {
+            selectHappySad(1);
+            return false;
+        });
+        $('#happy-button').keypress(function(ev) {
+            if ((ev.keyCode || ev.which) === '13') {
+                $('#happy-button').click();
+            }
+        });
```
Comment (rlr) [inline fjord/feedback/static/js/fxos_feedback.js]: nit: you could chain these two above. like:
```
    $('#happy-button')
        .click(function(ev) {
            selectHappySad(1);
            return false;
        })
        .keypress(function(ev) {
            if ((ev.keyCode || ev.which) === '13') {
                $('#happy-button').click();
            }
        });
```

It's mostly a styling preference but it saves a dom call to get the node.

File: fjord/feedback/templates/feedback/fxos_feedback.html [hunk 4]
```diff
@@ -13,221 +8,264 @@
...
+            </div>
+
+            <div id="sad-button" class="sentiment-button" tabindex=0>
+              <svg class="face sad-face" viewBox="0 0 100 100">
+                <g>
+                  <!-- 100 by 100 -->
+                  <circle cx="50" cy="50" r="43" stroke-width="7" fill="none"/>
+                  <circle cx="33" cy="37" r="7" stroke-width="0" />
+                  <circle cx="67" cy="37" r="7" stroke-width="0" />
+                  <path d="M24 72 C 35 55, 65 55, 76 72" stroke-width="6" fill="none"/>
```
Comment (rlr) [inline fjord/feedback/templates/feedback/fxos_feedback.html]: i might play some of these numbers on the powerball.

**Comment by rlr** [comment]
I like the structure of the code. It all makes sense. I'm going to do a pass at testing now.

**Comment by rlr** [comment]
I went through the flow for Firefox and Firefox OS. Hit back a bunch of time from the page and from the browser back button. Didn't hit any issues. SHIP IT :ship: r+

File: fjord/feedback/static/js/fxos_feedback.js (hunk 3 above)
Comment (willkg) [inline fjord/feedback/static/js/fxos_feedback.js]: Oh, neato! Will change!

File: fjord/feedback/templates/feedback/fxos_feedback.html (hunk 4 above)
Comment (willkg) [inline fjord/feedback/templates/feedback/fxos_feedback.html]: Ha! If you win, you should submit some happy feedback.

**Pull request merged**